src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunBWAMEMViaCommandLineTest.java

```diff
@@ -28,24 +29,18 @@
     private static final Path bwaPath = getBwaPath();
     private static Path getBwaPath(){
         String s = System.getProperty("PATHTOBWA");
-        if(null==s){
-            s = getNonNullEnvironmentVariable("PATHTOBWA");
-            if(null==s) {
-                return null;
-            }else{
-                return Paths.get(s);
-            }
-        }else{
-            return Paths.get(s);
-        }
+        return s == null ? (null==(s=System.getenv("PATHTOBWA")) ? null : Paths.get(s)) : Paths.get(s);
```

Comment:
This is a little TOO compact! (hard to read). Can you move the `System.getenv("PATHTOBWA")` call back into a separate if statement before the `? :` line?